src/main/kotlin/no/nav/familie/ef/sak/vedtak/VedtakService.kt

```diff
@@ -100,6 +119,62 @@ class VedtakService(
     fun hentHarAktivtVedtak(behandlingId: UUID, localDate: LocalDate = LocalDate.now()): Boolean {
         return hentVedtak(behandlingId).erVedtakAktivtForDato(localDate)
     }
+
+    @Transactional
+    fun angreSendTilBeslutter(saksbehandling: Saksbehandling) {
+        val vedtak = hentVedtak(behandlingId = saksbehandling.id)
+
+        validerKanAngreSendTilBeslutter(saksbehandling, vedtak)
+
+        behandlingService.oppdaterStegPåBehandling(saksbehandling.id, steg = StegType.SEND_TIL_BESLUTTER)
+        behandlingService.oppdaterStatusPåBehandling(saksbehandling.id, status = BehandlingStatus.UTREDES)
+        behandlingshistorikkService.opprettHistorikkInnslag(
+            behandlingId = saksbehandling.id,
+            stegtype = saksbehandling.steg,
+            utfall = StegUtfall.ANGRE_SEND_TIL_BESLUTTER,
+            metadata = null
+        )
+
+        ferdigstillGodkjenneVedtakOppgave(saksbehandling)
+        opprettBehandleSakOppgave(saksbehandling)
+    }
+
+    private fun opprettBehandleSakOppgave(saksbehandling: Saksbehandling) {
+        taskService.save(
+            OpprettOppgaveTask.opprettTask(
+                OpprettOppgaveTask.OpprettOppgaveTaskData(
+                    behandlingId = saksbehandling.id,
+                    oppgavetype = Oppgavetype.BehandleSak,
+                    beskrivelse = "Angret send til beslutter"
+                )
+            )
+        )
+    }
+
+    private fun ferdigstillGodkjenneVedtakOppgave(saksbehandling: Saksbehandling) {
+        oppgaveService.hentOppgaveSomIkkeErFerdigstilt(oppgavetype = Oppgavetype.GodkjenneVedtak, saksbehandling)?.let {
+            taskService.save(
+                FerdigstillOppgaveTask.opprettTask(
+                    behandlingId = saksbehandling.id,
+                    oppgavetype = Oppgavetype.GodkjenneVedtak,
+                    it.gsakOppgaveId,
+                    personIdent = null
+                )
+            )
+        }
+    }
+
+    private fun validerKanAngreSendTilBeslutter(saksbehandling: Saksbehandling, vedtak: Vedtak) {
+        val innloggetSaksbehandler = SikkerhetContext.hentSaksbehandler()
```

Comment:
Når en operasjon kun skal brukes av en bruker så kan det være fint å bruke `strict = true` på `hentSaksbehandler`
```suggestion
        val innloggetSaksbehandler = SikkerhetContext.hentSaksbehandler(strict = true)
```